airbyte-ci/connectors_ci/metadata_service/orchestrator/orchestrator/assets/catalog_assets.py

```diff
@@ -0,0 +1,157 @@
+import pandas as pd
+import json
+
+from dagster import MetadataValue, Output, asset, OpExecutionContext
+
+from ..utils.html import html_body
+
+
+@asset(required_resource_keys={"catalog_report_directory_manager"})
+def connector_catalog_location_html(context, all_destinations_dataframe, all_sources_dataframe):
+    """
+    Generate an HTML report of the connector catalog locations.
+    """
+
+    columns_to_show = ["name_x", "dockerRepository", "dockerImageTag", "is_oss", "is_cloud"]
+
+    # convert true and false to checkmarks and x's
+    all_sources_dataframe.replace({True: "✅", False: "❌"}, inplace=True)
+    all_destinations_dataframe.replace({True: "✅", False: "❌"}, inplace=True)
+
+    title = "Connector Catalogs"
+    content = f"<h1>{title}</h1>"
+    content += "<h2>Sources</h2>"
+    content += all_sources_dataframe[columns_to_show].to_html()
+    content += "<h2>Destinations</h2>"
+    content += all_destinations_dataframe[columns_to_show].to_html()
+
+    html = html_body(title, content)
+
+    catalog_report_directory_manager = context.resources.catalog_report_directory_manager
+    file_handle = catalog_report_directory_manager.write_data(html.encode(), ext="html", key="connector_catalog_locations")
+
+    metadata = {
+        "gcs_path": MetadataValue.url(file_handle.gcs_path),
+    }
```

Comment:
nit: could be cool to use jinja templates for this usecase. I think it would help us have a single function to generate markdown or html outputs

Reply:
Hmm I like that, let me see about adding it